src/api/base_urls.tsx

```diff
@@ -0,0 +1 @@
+export const getWinesUrl = "http://localhost:4000/api/wines"
```

Comment:
Have you learned at all about API versioning?

Reply:
No, I have not. Never came across the term before.

Reply:
https://restfulapi.net/versioning/